.travis.yml

```diff
@@ -5,8 +5,8 @@ os:
   - osx
 cache: cargo
 sudo: false
-script: |
-  cargo test
-  pushd plugin
-  cargo test
-  popd
+script:
```

Comment:
Without this change, it seems that only the status code of the last command (popd) is used. If the 2nd `cargo test` failed, the build was passing, as it's status code was shadowed by the `popd` one.

Reply:
Interesting. Good catch :+1:!